### DISCUSSION TITLE
Neogeo parallel 1p 5v firmware

### DISCUSSION THREAD

**Discussion by raed143** [darthcloud/BlueRetro#472] [Q&A]
Anyone managed to install neogeo version? It work's really well. But it requires 2 different board. 1 for each controller. Do we use the same firmware for both? Noticed two different versions. 

If we use the same firmware both will indicate controller as controller 1. And sometime you just want to play single player game and the controller get paired as p2. It will be h
![20221124_152618](https://user-images.githubusercontent.com/95415129/203850116-25ffd110-0803-4406-980e-51538ad000ea.jpg)
elpful if there is a second version where it will show paired controller as p2 and delay pairing to ensure that first controller is assigned to p1. 
![20221124_152604](https://user-images.githubusercontent.com/95415129/203850132-fc0e5621-8ca2-48a7-bf5b-30a6090d6f55.jpg)
![20221124_152618](https://user-images.githubusercontent.com/95415129/203850177-2aa029ad-3c0d-44e2-b5cf-f1a28badaa3f.jpg)

**Comment by darthcloud** [comment]
The best way is to set Inquiry mode to manual (pairing mode) via web on both:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#22---global-config

Then you route both esp32 boot button externally and control which one you want to put in pairing mode.
https://github.com/darthcloud/BlueRetro/wiki#42---boot-io0-external-adapter

Once the controller are paired once they will reconnect on the right ESP32.
